### DISCUSSION TITLE
Is browser auto-reloading possible or planned?

### DISCUSSION THREAD

**Discussion by james0r** [uicrooks/shopify-theme-lab#106] [Q&A]
Just wanting to know if this is coming or whether this is something that's gonna be left up to the dev to implement.

Thanks!

**Comment by sergejcodes** [comment]
Auto-reloading should be possible in version 4 through the Shopify CLI in Chromium-based browsers. [Shopify docs](https://shopify.dev/themes/tools/cli/theme-commands#serve). In Theme Lab the `start` task utilizes the `shopify theme serve` command.
